Cargo.toml

```diff
@@ -23,7 +23,7 @@ futures-util = { version = "0.3", default-features = false, features = ["sink"]
 futures-channel = { version = "0.3.17", features = ["sink"]}
 headers = "0.3"
 http = "0.2"
-hyper = { version = "0.14", features = ["stream", "server", "http1", "http2", "tcp", "client"] }
+hyper = { version = ">=0.14.19", features = ["stream", "server", "http1", "http2", "tcp", "client"] }
```

Comment:
The `>=` isn't needed (in fact, it makes it accept ANY new version). Setting `0.14.19` will mean at least that, while still within `0.14.x`.

Reply:
Got it, fixed in 1bc098d32c678ddfbfedbb1cdda2992009cdc705, thanks